contracts/cryptoblades.sol

```diff
@@ -322,7 +316,7 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
             target,
             now / 1 hours
         );
-        uint256 offset = performFight(
+        uint256 expectedTokens = performFight(
```

Comment:
technically you could put the return clause here without the need for a variable initialization but this is fine too